src/arrays/ArrayUtilTest.java

```diff
@@ -4,5 +4,17 @@
 
     public static void main(String[] args) {
 
+                   ArrayUtil.createArray(4);
+                  ArrayUtil.create2dArray(2,3);
+                 ArrayUtil.addFirst(new int [] {5,4,8,1}, 2);
+                ArrayUtil.contains(new int [] {0,4,5,6,8}, 5);
+               ArrayUtil.indexOf(new int [] {25,1,45,7,0,9}, 0);
+              ArrayUtil.remove (new int[] {4,5,6,2,5,8}, 2);
+             ArrayUtil.sum(new int [] {5,8,9,21,1});
+            ArrayUtil.getMax (new int [] {5,69,12,89,25,90,0});
+           ArrayUtil.getMin(new int [] {24,5,15,64,-5,1});
+          ArrayUtil.getAvg(new int [] {5,6,7,8,9,1});
```

Comment:
In order not to create a new array for each method, you can create an array in the ArrayUtilTest class and call it.

similar to this

static int[] array = {1, 5, 6, 7, 8};  and call array